gradle.properties

```diff
@@ -50,7 +50,7 @@ jodaTimeVersion=2.9
 reflectionsVersion=0.9.10
 guavaVersion=18.0
 
-pac4jVersion=1.8.0
+pac4jVersion=1.8.1-SNAPSHOT
```

Comment:
pac4j v1.8.1 will be released end November / begin December. I'll update the version after that.